lib/net/ldap.rb

```diff
@@ -1286,11 +1286,9 @@ def use_connection(args)
     else
       begin
         conn = new_connection
-        if (result = conn.bind(args[:auth] || @auth)).result_code == Net::LDAP::ResultCodeSuccess
-          yield conn
-        else
-          return result
-        end
+        result = conn.bind(args[:auth] || @auth)
+        return result unless result.code == Net::LDAP::ResultCodeSuccess
```

Comment:
rubocode failed to convert codes. `result_code` should be called here, not `code`. All tests would be passed by fixing it.  @davesims @jch @tmaher 😃